src/Sidebar/Sidebar.stories.js

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import Sidebar from "./Sidebar";
+
+const Template = (args) => <Sidebar {...args} />;
+
+export const Default = Template.bind({});
+
+Default.args = {
+  activeTab: "scheduled_task",
+  tabs: [
+    {
+      id: "workflow",
+      title: "Workflow",
+      icon: "calendar",
+      nested: [
+        {
+          title: "Scheduled Tasks",
+          id: "scheduled_task",
+          link: () => alert("Scheduled Tasks clicked"),
```

Comment:
this is triggering me lol

```suggestion
          id: "scheduled_task",
          title: "Scheduled Tasks",
          link: () => alert("Scheduled Tasks clicked"),
```